docs/data-sources/remote_docker_repository.md

```diff
@@ -0,0 +1,29 @@
+---
+subcategory: "Remote Repositories"
+---
+# Artifactory Remote Docker Repository Data Source
+
+Retrieves a remote Docker repository.
+
+## Example Usage
+
+```hcl
+data "artifactory_remote_docker_repository" "remote-docker" {
+  key = "remote-docker"
+}
+```
+
+## Argument Reference
+
+The following argument is supported:
+
+* `key` - (Required) the identity key of the repo.
+
+## Attribute Reference
+
+The following attributes are supported, along with the [common list of attributes for the remote repositories](remote.md):
+
+* `block_pushing_schema1` - (Optional) When set, Artifactory will block the pulling of Docker images with manifest v2 schema 1 from the remote repository (i.e. the upstream). It will be possible to pull images with manifest v2 schema 1 that exist in the cache.
+* `enable_token_authentication` - (Optional) Enable token (Bearer) based authentication.
+* `external_dependencies_enabled` - (Optional) Also known as 'Foreign Layers Caching' on the UI.
+* `external_dependencies_patterns` - (Optional) An allow list of Ant-style path patterns that determine which remote VCS roots Artifactory will follow to download remote modules from, when presented with 'go-import' meta tags in the remote repository response. By default, this is set to '[**]' in the UI, which means that remote modules may be downloaded from any external VCS source. Due to SDKv2 limitations, we can't set the default value for the list. This value '[**]' must be assigned to the attribute manually, if user don't specify any other non-default values. We don't want to make this attribute required, but it must be set to avoid the state drift on update. Note: Artifactory assigns '[**]' on update if HCL doesn't have the attribute set or the list is empty.
```

Comment:
This is how it will look in the TF documentation. 
<img width="665" alt="image" src="https://user-images.githubusercontent.com/55418063/223532961-b928756d-0bb1-4768-bcfb-5b5a7f72d813.png">
' should be replaced by `
We also need to fix it in the other 7 places.